src/commonMain/kotlin/exchange.dydx.abacus/state/manager/StateManagerAdaptor.kt

```diff
@@ -1880,10 +1878,15 @@ open class StateManagerAdaptor(
         val cancelOrderPayloads = mutableListOf<HumanReadableCancelOrderPayload>()
         val triggerOrders = stateMachine.state?.input?.triggerOrders
 
+        val marketId = triggerOrders?.marketId ?: throw Exception("marketId is null")
```

Comment:
kotlin tip: we can instead enforce `triggerOrders` is non-null by doing:
`val triggerOrders = requireNotNull(stateMachine.state?.input?.triggerOrders) { "triggerOrders input was null" }`

then we don't need to do null checks on it later.

Reply:
part of me wants to write a codemod to go through the whole repo to replace all of the `throwException`s